content/pl/concepts/overview/components.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- lavalamp
+title: Składniki Kubernetes
+content_template: templates/concept
+weight: 20
+card: 
+  name: concepts
+  weight: 20
+---
+
+{{% capture overview %}}
+W wyniku instalacji Kubernetes otrzymujesz klaster.
+
+{{< glossary_definition term_id="cluster" length="all" prepend="Klaster to">}}
+
+W tym dokumencie opisujemy składniki niezbędne do zbudowania kompletnego, poprawnie działającego klastra Kubernetes.
+
+Poniższy rysunek przedstawia klaster Kubernetes i powiązania pomiędzy jego różnymi częściami składowymi.
+
+![Składniki Kubernetes](/images/docs/components-of-kubernetes.png)
+{{% /capture %}}
+
+{{% capture body %}}
+## Master — częsci składowe
+
+Komponenty *master* odpowiadają za warstwę sterowania klastra. Podejmują ogólne decyzje dotyczące klastra (np. zlecanie zadań), wykrywają i reagują na zdarzenia w klastrze (przykładowo, start nowego {{< glossary_tooltip text="poda" term_id="pod">}}, kiedy wartość `replicas` dla deploymentu nie zgadza się z faktyczną liczbą replik).
+
+Komponenty *master* mogą być uruchomione na dowolnej maszynie w klastrze. Dla uproszczenia skrypty instalacyjne zazwyczaj startują wszystkie składniki na tej samej maszynie i jednocześnie nie pozwalają na uruchamianie na niej kontenerów użytkowników. Na stronie [Building High-Availability Clusters](/docs/admin/high-availability/) jest więcej informacji o konfiguracji typu *multi-master-VM*.
+
+### kube-apiserver
+
+{{< glossary_definition term_id="kube-apiserver" length="all" >}}
+
+### etcd
+
+{{< glossary_definition term_id="etcd" length="all" >}}
+
+### kube-scheduler
+
+{{< glossary_definition term_id="kube-scheduler" length="all" >}}
+
+### kube-controller-manager
+
+{{< glossary_definition term_id="kube-controller-manager" length="all" >}}
+
+Kontrolerami są:
+
+* Node Controller: Odpowiada za rozpoznawanie i reagowanie na sytuacje, kiedy węzeł staje się z jakiegoś powodu niedostępny.
+* Replication Controller: Odpowiada za utrzymanie prawidłowej liczby podów dla każdego obiektu typu *replication controller* w systemie.
```

Comment:
One nit:
```suggestion
* Replication Controller: Odpowiada za utrzymanie prawidłowej liczby podów dla każdego obiektu typu *ReplicationController* w systemie.
```
because the object type is ReplicationController; in English the difference is small whereas in Polish it'd be something like “kontroler replikacja” vs ReplicationController and there's less similarity.